Makefile

```diff
@@ -24,38 +24,16 @@ help:
 	@echo ""
 	@echo "  Variables"
 	@echo ""
-	@echo "    OLENA_VERSION  Olena version to use ('$(OLENA_VERSION)')"
 	@echo "    PREFIX         directory to install to ('$(PREFIX)')"
 	@echo "    PYTHON         Python binary to bind to ('$(PYTHON)')"
 	@echo "    PIP            Python pip to install with ('$(PIP)')"
 
 # END-EVAL
 
-# Olena version to use
-#OLENA_VERSION ?= 2.1
-OLENA_VERSION ?= git
+OLENA_DIR = olena-git
```

Comment:
Why keep a variable for this if the path is fixed in .gitmodules anyway, and there is no way to use the tarball anymore?

Reply:
To reduce the number of changes and for consistency.